igor_src/Aligner.cpp

```diff
@@ -251,6 +251,93 @@ forward_list<Alignment_data> Aligner::align_seq(string nt_seq , double score_thr
 	return alignment_list;
 }
 
+
+forward_list<Alignment_data> Aligner::align_seq(string nt_seq , double score_threshold , bool best_only , unordered_map<string,pair<int,int>> genomic_offset_bounds,bool rev_offset_frame){
+	int min_offset;
+	int max_offset;
+	Int_Str int_seq = nt2int(nt_seq);
+        int seqlen=int_seq.size();
+	forward_list<Alignment_data> alignment_list;// = *(new forward_list<Alignment_data>());
+	for(forward_list<pair<string,Int_Str>>::const_iterator iter = int_genomic_sequences.begin() ; iter != int_genomic_sequences.end() ; iter++){
+	        min_offset=-genomic_offset_bounds.at((*iter).first).first-3;  
```

Comment:
Woups had not seen that before merging, what are these -3 and +3 about?
Maybe I'm confused but I feel these are here to compensate for an error you have made upstream. At this medium level of alignments (Aligner::align_seq) there should not be any reference to aa or nt (so no +/-3 codon issue)